### ISSUE TITLE
Introduce new technical exercise for git workflows

### ISSUE THREAD

**Issue by oybed** [rht-labs/enablement-docs#347]
**Is your feature request related to a problem? Please describe.**
Many participants in this course are new to git, and this leads to confusion and uncertainty. Since all technical exercises utilizes git, some participants may loose out on the main exercise context as the unfamiliarity with git gets in the way. 

**Describe the solution you'd like**
I think it would be good to introduce a new, **small**, technical exercise that covers git basics and some git workflows. This could be a small follow-up to exercise 1, or maybe even just part of exercise 1. This could then cover git practices around: forking, branching, remote v.s. local, pull/rebase/merge, etc.

**Describe alternatives you've considered**
Make "learning git" a prereq to taking the course. However, not sure that will work as people in general are busy and cannot complete the prereq in time. 

**Additional context**
See issue #338 and #346 for some additional context

**Comment by springdo** [comment]
Hi @oybed  - there is a git pre-req in the instructions which is suggested all do. I believe it's also on the email sent to the attendees to complete it before they attend the course. 
However if this is not enough in your eyes, it only takes 15 mins and could be run with the groups as a pre-step to exercise 1 if needs be.

**Comment by oybed** [comment]
@springdo yes, please see my "Alternatives" comment in original write-up. In general, multiple participants do not find the time to do the prereqs. + I suspect that non-technical people would find it beneficial to discussed and ask questions from an instructor around this topic. I believe this could be a good topic for a mobbing activity.

**Comment by springdo** [comment]
Yeah man! That sounds cool, I like the ideas of having the groups / teams do it as a mob activity!

**Comment by gsampaio-rh** [comment]
We can also use that to explain different branch strategies like Gitflow, Github Flow, Trunk Based, One Flow...

Maybe during the presentation or/and during the exercises.

**Comment by springdo** [comment]
This issue would also relate to this one: https://github.com/rht-labs/enablement-docs/issues/123 I I will close trunk  issue in favour of this one.
